analysis/processor/process.go

```diff
@@ -0,0 +1,66 @@
+package processor
+
+import (
+	"fmt"
+
+	"github.com/deepsourcelabs/cli/types"
+)
+
+type fileContentNode struct {
+	Filename    string
+	FileContent []string
+}
+
+// Processor interface to receive analysis post-processors.
+type IProcessor interface {
+	Name() string
+	Process([]string, *types.Issue, *[]types.Issue) error
+}
+
+// ProcessAnalysisResults struct contains the data needed to process the analysis results.
+type ProcessAnalysisResults struct {
```

Comment:
Changed it to `ConfigureProcessors` and reduced the number of struct elements to just the ones required to configured the processor. Have removed the `AnalysisResult` and `processedIssues` since they had no relation with configuring processors. 

Ref: https://github.com/deepsourcelabs/cli/blob/plt-4734-plt-4735-add-post-processors-support/analysis/processor/process.go#L16